app/connectors/FileNotificationOrchestratorConnector.scala

```diff
@@ -29,8 +29,7 @@ class FileNotificationOrchestratorConnector @Inject()(httpClient: HttpClient,
                                                      (implicit ec: ExecutionContext) {
 
   def postFileNotifications(notifications: Seq[SDESNotification])(implicit hc: HeaderCarrier): Future[HttpResponse] = {
-    val headerCarrierWithInternalAuthToken = hc.copy(authorization = Some(Authorization(appConfig.internalAuthBearerToken)))
     httpClient.POST[Seq[SDESNotification], HttpResponse](url = appConfig.postFileNotificationUrl, notifications)(
-      SDESNotification.seqOfWrites, Implicits.readRaw, headerCarrierWithInternalAuthToken, implicitly)
+      SDESNotification.seqOfWrites, Implicits.readRaw, hc, implicitly)
```

Comment:
As all the arguments are implicit, you should be able to remove the second pair of brackets.
```suggestion
```